Makefile:204: recipe for target 'shellcheck' failed

* Problem:
Below error when run command "sudo make" : 
recipe for target 'shellcheck' failed 

* Reproduce:
$ git clone prometheus-operator
$ cd prometheus-operator
$ kubectl apply -f bundle.yaml
$ sudo make

* Environment:
Ubuntu 18.04 
Kubernetes 1.15
DIND

* Prometheus Operator version: 0.31.1
  $ sudo kubectl -n monitoring describe deployment prometheus-operator
  Error from server (NotFound): deployments.extensions "prometheus-operator" not found

* Kubernetes version information:
$ sudo kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.1", GitCommit:"4485c6f18cee9a5d3c3b4e523bd27972b1b53892", GitTreeState:"clean", BuildDate:"2019-07-24T18:15:39Z", GoVersion:"go1.12.7", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:02:58Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}


* Kubernetes cluster kind:
dind

* Prometheus Operator Logs:
~/prometheus-operator$ sudo make
go fmt github.com/coreos/prometheus-operator/cmd/lint github.com/coreos/prometheus-operator/cmd/operator github.com/coreos/prometheus-operator/cmd/po-crdgen github.com/coreos/prometheus-operator/cmd/po-docgen github.com/coreos/prometheus-operator/cmd/po-rule-migration github.com/coreos/prometheus-operator/cmd/prometheus-config-reloader github.com/coreos/prometheus-operator/example/alertmanger-webhook github.com/coreos/prometheus-operator/pkg/admission github.com/coreos/prometheus-operator/pkg/alertmanager github.com/coreos/prometheus-operator/pkg/api github.com/coreos/prometheus-operator/pkg/apis/monitoring github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1 github.com/coreos/prometheus-operator/pkg/client/informers/externalversions github.com/coreos/prometheus-operator/pkg/client/informers/externalversions/internalinterfaces github.com/coreos/prometheus-operator/pkg/client/informers/externalversions/monitoring github.com/coreos/prometheus-operator/pkg/client/informers/externalversions/monitoring/v1 github.com/coreos/prometheus-operator/pkg/client/listers/monitoring/v1 github.com/coreos/prometheus-operator/pkg/client/versioned github.com/coreos/prometheus-operator/pkg/client/versioned/fake github.com/coreos/prometheus-operator/pkg/client/versioned/scheme github.com/coreos/prometheus-operator/pkg/client/versioned/typed/monitoring/v1 github.com/coreos/prometheus-operator/pkg/client/versioned/typed/monitoring/v1/fake github.com/coreos/prometheus-operator/pkg/k8sutil github.com/coreos/prometheus-operator/pkg/listwatch github.com/coreos/prometheus-operator/pkg/prometheus github.com/coreos/prometheus-operator/pkg/version
./scripts/check_license.sh
docker run -v ":/mnt" koalaman/shellcheck:stable ./scripts/create-minikube.sh ./scripts/travis-push-docker-image.sh ./scripts/check_license.sh ./scripts/generate-bundle.sh ./scripts/generate/build-rbac-prometheus-operator.sh ./scripts/generate/build-non-rbac-prometheus-operator.sh ./scripts/delete-minikube.sh ./scripts/travis-e2e.sh ./scripts/run-external.sh
./scripts/create-minikube.sh: ./scripts/create-minikube.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/travis-push-docker-image.sh: ./scripts/travis-push-docker-image.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/check_license.sh: ./scripts/check_license.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/generate-bundle.sh: ./scripts/generate-bundle.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/generate/build-rbac-prometheus-operator.sh: ./scripts/generate/build-rbac-prometheus-operator.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/generate/build-non-rbac-prometheus-operator.sh: ./scripts/generate/build-non-rbac-prometheus-operator.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/delete-minikube.sh: ./scripts/delete-minikube.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/travis-e2e.sh: ./scripts/travis-e2e.sh: openBinaryFile: does not exist (No such file or directory)
./scripts/run-external.sh: ./scripts/run-external.sh: openBinaryFile: does not exist (No such file or directory)
Makefile:204: recipe for target 'shellcheck' failed
make: *** [shellcheck] Error 2

--- a/Makefile
+++ b/Makefile
@@ -201,7 +201,7 @@ check-license:
 
 .PHONY: shellcheck
 shellcheck:
-	docker run -v "${PWD}:/mnt" koalaman/shellcheck:stable $(shell find . -type f -name "*.sh" -not -path "*vendor*")
+	docker run -v "$(PWD):/mnt" koalaman/shellcheck:stable $(shell find . -type f -name "*.sh" -not -path "*vendor*")
 
 ###########
 # Testing #

